application/org.openjdk.jmc.browser.attach/src/main/java/org/openjdk/jmc/browser/attach/LocalJVMToolkit.java

```diff
@@ -373,6 +373,13 @@ public DiscoveryEntry call() throws Exception {
 						// This leaks one thread handle due to Sun bug in j2se/src/windows/native/sun/tools/attach/WindowsVirtualMachine.c
 						Properties props = null;
 						try {
+							try {
+								// try to force finish init the attached JVM
```

Comment:
Would be great to add "JMC-4454:" here, so that people can go search for context if needed.